docs/topics/jvm/jvm-records.md

```diff
@@ -0,0 +1,74 @@
+[//]: # (title: Records)
+
+> JVM records are [Experimental](components-stability.md). They may be dropped or changed at any time.
+> Opt-in is required (see the details [below](#enabling-jvm-records)), and you should use them only for evaluation purposes. We would appreciate your feedback on them in [YouTrack](https://youtrack.jetbrains.com/issue/KT-42430).
+>
+{type="warning"}
+
+_Records_ are [classes](https://openjdk.java.net/jeps/395) in Java for storing immutable data. Records carry a fixed set of values – the _records components_.
+They have a concise syntax in Java and save you from having to write boilerplate code:
+
+```java
+// Java
+public record Person (String name, int age) {}
+```
+
+The compiler automatically generates a final class inherited from [java.lang.Record](https://download.java.net/java/early_access/jdk16/docs/api/java.base/java/lang/Record.html) with the following members:
```

Comment:
```suggestion
The compiler automatically generates a final class inherited from [`java.lang.Record`](https://download.java.net/java/early_access/jdk16/docs/api/java.base/java/lang/Record.html) with the following members:
```

Reply:
That one is tricky – I usually try not to use more than one logical markup element. I think there is room for discussion :)
However, I'll approve your suggestion.